src/wanna/core/services/notebook.py

```diff
@@ -696,14 +694,14 @@ def sync(self) -> None:
         if to_be_created:
             to_be_created_str = "\n".join(["-" + item.name for item in to_be_created])
             logger.user_info(f"Managed notebooks to be created:\n{to_be_created_str}")
-            should_create = typer.confirm("Are you sure you want to create them?")
-            if should_create:
+            should_create = typer.confirm("Are you sure you want to create them?") if not force else False
```

Comment:
I would do `should_create = True if force else typer.confirm("Are you sure you want to create them?")`

and then remove the `force or` in the next line

same for should_delete

WDYT

Reply:
Nice logic optimisation :) I will correct